gtests/net/packetdrill/packetdrill.c

```diff
@@ -61,12 +61,25 @@ static void run_init_scripts(struct config *config)
 	free(scripts);
 }
 
+/* Sets "PACKETDRILL_PID" environment variable to pid of this process,
```

Comment:
The PACKETDRILL_PID changes in here seem to be unrelated to the stated subject of the commit. In the Linux git ecosystem we try to keep commits as small and narrowly-focused as possible, and we try to accomplish things, where possible, using clean cherry-picks of existing commits elsewhere.

Please pull out the PACKETDRILL_PID diffs with something like the following:

git checkout packetdrill/master
git cherry-pick <internal_PACKETDRILL_PID_sha1>
 < clean up internal footers, leaving Change-Id, and add signed-off-by >
git cherry-pick 733f92fa5ddd25aea76d2d6d655277dfff0cfc93

Reply:
done. split into two commtis.